src/Components/Bookshelf/Bookshelf.tsx

```diff
@@ -1 +1,24 @@
-export {}
\ No newline at end of file
+import React from 'react'
+import '../Book/Book'
+
+type BookshelfProps = {
+  queryID: string
+}
+
+class Bookshelf extends React.Component<BookshelfProps> {
+  constructor(props: BookshelfProps) {
+    super(props)
+    this.state= {
+
+    }
+  }
+
+  render() {
+    return (
+      <h1>Hello</h1>
+    )
+  }
+}
+
+
+export default Bookshelf
```

Comment:
Good idea creating this as a class component, I had a feeling we'd use state in bookshelf to hold favorites.